src/components/PeopleTablet/PeopleTablet.tsx

```diff
@@ -0,0 +1,77 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleTablet: React.FC<Props> = ({ people }) => {
+  const { slug: selected } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug,
+          } = person;
+
+          const mother: Person | undefined = people
+            .find(prsn => prsn.name === motherName);
```

Comment:
```suggestion
          const mother = people
            .find(prsn => prsn.name === motherName);
```
There is actually no need to specify the type. Also, you could set '-' as the reserve value